explorer/src/Pos/Explorer/Web/ClientTypes.hs

```diff
@@ -100,9 +101,11 @@ import           Servant.API                      (FromHttpApiData (..))
 --
 -- The following types explain the situation better:
 --
--- type AddressHash = AbstractHash Blake2b_224
--- type Hash        = AbstractHash Blake2b_256
--- type TxId        = Hash Tx
+-- type AddressHash   = AbstractHash Blake2b_224
+-- type Hash          = AbstractHash Blake2b_256
+--
+-- type TxId          = Hash Tx               = AbstractHash Blake2b_256 Tx
+-- type StakeholderId = AddressHash PublicKey = AbstractHash Blake2b_224 PublicKey
```

Comment:
How would you update this comment with these comment if the original types are changed by anybody else?

Reply:
Manually. This is the simplest thing I could do to explain to myself and others what are these things, and that is pretty valuable.
I personally don't think it's self-explanatory and these comments do simplify reading the code.

Reply:
I see your point and the comment is a nice documentation. However, nobody will care about this comment if - for example - `StakeholderId` has been changed and the type in this comment might be deprecated then. Just because `StakeholderId` is defined in `Pos.Core.Address` and not here in `Pos/Explorer/Web/ClientTypes.hs`. In such a case the comment will not just be useless but also wrong. We have already this case with deprecated `ssc` types. Maybe you could add some fictional type to explain it and link to the modules of these types to have real examples. Just an idea.

Reply:
> Maybe you could add some fictional type to explain it and link to the modules of these types to have real examples. Just an idea.

It's a good idea. I will think how I to use types to show this information. 
If you want them removed, I can remove them, but I would wait until we see a single one of them is wrong (which I don't think will happen soon).